src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.et.syaapi.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.et.syaapi.service.CaseService;
+
+import javax.validation.constraints.NotNull;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@Slf4j
+@RequiredArgsConstructor
+@RestController
+public class ManageCaseController {
+
+    @Autowired
+    private CaseService caseService;
+
+    @GetMapping("/caseDetails/{caseId}")
+    @Operation(summary = "Return case details")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "403", description = "Calling service is not authorised to use the endpoint"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CaseDetails> getCaseDetails(
+        @RequestHeader("Authorization") String authorization,
+        @PathVariable String caseId
+    ) {
+        var caseDetails = caseService.getCaseData(authorization, caseId);
+        return ok(caseDetails);
+    }
+
+    @PostMapping("/case-type/{caseType}/event-type/{eventType}/case")
```

Comment:
Do we need to force the client to know about caseType and eventType?

Reply:
CaseType I think the client would need to know about to determine if its EnglandWales or Scotland. EventType could probably be removed in favor of a stand alone draft case creation endpoint. What do you think @jthmcts?

Reply:
We will look to update this endpoint once we integrate the postcode lookup in the create case service